lib/src/app/app.dart

```diff
@@ -44,11 +171,66 @@ class AppClient {
     return response.location;
   }
 
-  /// List the [Robot] of a specific [Location] that the currently authenticated user has access to
-  Future<List<Robot>> listRobots(Location location) async {
-    final listRobotsRequest = ListRobotsRequest()..locationId = location.id;
-    final ListRobotsResponse response = await _client.listRobots(listRobotsRequest);
-    return response.robots;
+  /// Update a [Location]
+  Future<Location> updateLocation(String locationId, {String? name, String? parentLocationId, String? region}) async {
+    final request = UpdateLocationRequest()
+      ..locationId = locationId
+      ..name = name ?? ''
+      ..parentLocationId = parentLocationId ?? ''
+      ..region = region ?? '';
```

Comment:
I think, similar to `updateOrganization`, we want to not set these values if they're not provided.